verilog/tools/ls/symbol-table-handler.h

```diff
@@ -65,11 +65,18 @@ class SymbolTableHandler {
       const verible::lsp::ReferenceParams &params,
       const verilog::BufferTrackerContainer &parsed_buffers);
 
+  verible::lsp::Range FindRenameLocations(
```

Comment:
These new fucntions need a test in symbol-table-handler_test.cc. These functions are implicitly tested later in the verilog-language-server_test.cc, but in the symbol-table-handler_test.cc, we should check all the potential corner cases while the verilog-language-sever_test.cc is more for the final end2end testing.

An indication about the tests you can get by running the coverage by just running the specific `symbol-table-handler_test.cc` to see how much it covers from `symbol-table.cc`:

```
MODE=coverage .github/bin/build-and-test.sh verilog/tools/ls:symbol-table-handler_test
.github/bin/generate-coverage-html.sh
# Then open browser and check out the coverage of symbol-table-handler.cc
```

... Our goal is typically to be > 90% for these library specific tests.

Reply:
The new methods in this PR are now covered in `symbol-table-handler_test`